e2e/testim/client.go

```diff
@@ -84,6 +84,7 @@ func (t *Client) NewRun(kubeconfig string, test inventory.Test, runOptions RunOp
 			args,
 			"--tunnel",
 			fmt.Sprintf("--tunnel-port=%s", runOptions.TunnelPort),
+			fmt.Sprintf("--base-url=http://localhost:%s", runOptions.TunnelPort), // TODO: this is a temportary workaround for a testim issue with ngrok tunnels
```

Comment:
This is usually not necessary as it's set by testim CLI automatically when using dedicated tunnels: https://help.testim.io/docs/dedicated-run-tunnel. However, given the ongoing issue, they have provided this to us as a temporary workaround.